### PR TITLE
Delete dqm muon alignment

### PR DESCRIPTION
**Pull request by jfernan2**
#### PR description:

DQMOffline/Alignment/ MuonAlignment module not been used any longer
The change affects to all AlCaRecos starting with pathALCARECOMuAl* which were producing DQM plots with this module but are NOT needed at present

#### PR validation:

Tested several AlCarecos:
runTheMatrix.py -l 10006.0,10806.0,10042.0,1308.0,1307.0,7.23,7.0,4.22,1004.0 -i all --ibeos

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

No backport foreseen

This PR tries to fix the following issues 
https://github.com/cms-sw/cmssw/issues/28975 
and 
https://github.com/cms-sw/cmssw/issues/28974

### CODE REVIEW

**Comment by jfernan2** [comment]
please test wf 10006.0,10806.0,10042.0,1308.0,1307.0,7.23,7.0,4.22,1004.0

**Comment by jfernan2** [comment]
please test 10006.0,10806.0,10042.0,1308.0,1307.0,7.23,7.0,4.22,1004.0

**Comment by jfernan2** [comment]
please test workflow 10006.0,10806.0,10042.0,1308.0,1307.0,7.23,7.0,4.22,1004.0

**Comment by makortel** [comment]
Thanks @jfernan2!

**Comment by jfernan2** [comment]
+1
Plots removed consistent with expectations. Empty in many cases

File: DQMOffline/Alignment/src/SealModule.cc [hunk 1]
```diff
@@ -1,9 +1,2 @@
 #include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/PluginManager/interface/ModuleDef.h"
-
-// the clients
-#include "DQMOffline/Alignment/interface/MuonAlignment.h"
-#include "DQMOffline/Alignment/interface/MuonAlignmentSummary.h"
-
-DEFINE_FWK_MODULE(MuonAlignment);
-DEFINE_FWK_MODULE(MuonAlignmentSummary);
```
Comment (schneiml) [inline DQMOffline/Alignment/src/SealModule.cc]: I think this file can be completely removed now.

Comment (jfernan2) [reply DQMOffline/Alignment/src/SealModule.cc]: Done

**Comment by jfernan2** [comment]
+1

**Comment by silviodonato** [comment]
This PR is removing all plots in AlCaReco in wf 1000.0.
Was this expected?

**Comment by silviodonato** [comment]
@jfernan2

**Comment by silviodonato** [comment]
I see from https://github.com/cms-sw/cmssw/issues/28975#issuecomment-588483569 that this was actually expected

**Comment by silviodonato** [comment]
assign alca

**Comment by silviodonato** [comment]
@christopheralanwest,@franzoni,@tlampen,@pohsun,@tocheng we are removing the module DQMOffline/Alignment/MuonAlignment from CMSSW. This module is used in wf 1000.0. Please note this module different from Alignment/MuonAlignment/MuonAlignment .
As @jfernan2 wrote in #28975 (comment), this module is not longer used for muon alignment.
This PR solves #28975 and #28974.

**Comment by silviodonato** [comment]
+operations

**Comment by tocheng** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Comment by jfernan2** [comment]
@silviodonato sorry if I did not spread the news enough: I got green from AlCa and Muon Alignment group before creating the PR, which is not emoving ALL AlCa plots but those produced (for historical reasons) in the Muon Alignment DQM which are at present empty and not used.
Thanks for checking anyway!

**Comment by mmusich** [comment]
once upon a time having an AlCa DQM module to monitor what was actually produced in the ALCARECOs was a `conditio sine qua` non to actually get the green light for production. 
I am wondering what has changed in the policy.

**Pull request merged**